integrations/cohere/examples/cohere_generation.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
```

Comment:
I think adding a brief comment (and/or some docs links) explaining this example would be beneficial for users unfamiliar with `BranchJoiner` and `JsonSchemaValidator`.

Reply:
Deal, will do

Reply:
Have a brief look at [4b1bee8](https://github.com/deepset-ai/haystack-core-integrations/pull/838/commits/4b1bee8df99d01d5fc1620d2c8a18391a33efda5) @anakin87

Reply:
Nice!